### ISSUE TITLE
redirect on current tab when invidious tab video has an error

### ISSUE THREAD

**Issue by rezad1393** [libredirect/libredirect#66]
redirect on current tab when invidious tab video has an error.

when I open a youtube link that has as error (for example adult flagged video so need login or not loading for other reasons)
then the invidious tab seem to try to refresh again and again untill error in gone. I think this is from this addon because I have set only one invidious instance.
so far that is not bad.
but the issue is that when that refresh (technically redirect) happens, if I switch to another tab then that tab gets redirected to the invidious url.
this seems to be a bug (kinda serious if you ask me) in handling of tab and urls in this addon.

right now I dont have a repeatable url for youtube to test this because the ones I has encountered for this bug , after manual refresh of page they load fine.

please check the addon code for this redirect and fix this.

it may be an internal logic of invidious itself that refresh the page but because when I use the older addon (privacy redirect) I dont get this bug I think this is the libreredirect issue.

**Comment by ManeraKai** [comment]
which version are you running? all auto redirections of offline instances are now manual. That is the only think I can think of.

**Comment by ManeraKai** [comment]
> then the invidious tab seem to try to refresh again and again until error in gone

This specifically was because of auto-redirection if the instance was offline. I converted changing instances from auto to a manual button in the popup.

I don't think you will encounter this problem again as there's nothing automatically in the extension right now other than redirecting from the original main sites like `https://youtube.com`.

**Comment by ManeraKai** [comment]
Have you encountered this problem again in those 8 days?

**Comment by rezad1393** [comment]
when I reported that error I had more time and I was watching a lot of youtube backlog.
even then the error was not reproducible because when the invidious instance has a loading error and this happened, it usually got fixed by F5 from firefox.
maybe removing the addon and reinstalling it ( to fix another bug I reported) fixed this?
maybe close this and I will reopen it if it happens again and I remember?
